deps/jshttp/mime-db/v1.37.0/db.json

```diff
@@ -0,0 +1,7688 @@
+{
```

Comment:
I don't think we need this deps dir - or at least it's something that can be added later when we have more than a single dep.
Just place the `db.json into `//media_types`

Reply:
👍